src/routes/people/+page.svelte

```diff
@@ -12,25 +16,89 @@
     import eyronTanael from '$lib/people/eyron-tanael.webp';
     import hansLorico from '$lib/people/hans-lorico.webp';
     import jellyRaborar from '$lib/people/jelly-raborar.webp';
+    import lino from '$lib/lino-hero.svg';
+
+    import { get } from '$lib/stores/color-scheme';
+
+    let pastExec = false;
+    const theme = get();
 </script>
 
-<article class="prose flex max-w-none flex-col gap-8">
-    <figure class="m-0 flex flex-col items-center">
-        <picture class="max-w-3xl">
-            <source type="image/webp" srcset={execSmall} media="(max-width:640px)" />
-            <source type="image/webp" srcset={execMedium} />
-            <img src={execFull} width="1568" height="1203" alt="The Executive Board" />
-        </picture>
-        <figcaption>The 2023-2024 Executive Board</figcaption>
-    </figure>
-    <section class="not-prose flex flex-wrap justify-around gap-8">
-        <Exec src={calebBunye} alt="Caleb Bunye" pos="President" />
-        <Exec src={hansLorico} alt="Hans Lorico" pos="Vice President for Innovation" />
-        <Exec src={eyronTanael} alt="Eyron Tanael" pos="Vice President for Service" />
-        <Exec src={bastiOrtiz} alt="Basti Ortiz" pos="Director for Engineering" />
-        <Exec src={arianRey} alt="Arian Rey" pos="Director for Membership & Internals" />
-        <Exec src={jellyRaborar} alt="Jelly Raborar" pos="Director for Branding & Creatives" />
-        <Exec src={beaDiokno} alt="Bea Diokno" pos="Director for External Relations" />
+<section>
+    <section class="prose mb-8 flex max-w-none flex-col gap-8">
+        <figure class="m-0 flex flex-col items-center">
+            <!-- <picture class="max-w-3xl">
+                <source type="image/webp" srcset={execSmall} media="(max-width:640px)" />
+                <source type="image/webp" srcset={execMedium} />
+                <img src={execFull} width="1568" height="1203" alt="The Executive Board" />
+            </picture> -->
```

Comment:
The point of Git is the ability to go back in time without worrying about our code getting deleted forever. Commenting out code is common practice in codebases without version control, but since we're using Git here, let's fearlessly delete unused code. 🚀